### PR TITLE
feat: use fetch

### PR DESCRIPTION
**Pull request by oed**
This should be simple enough. Please test locally with example for various scenarios of how the address-server is used.

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -46,34 +46,18 @@ module.exports = {
...
-        }
-      }
-      request.open(method, url)
-      request.setRequestHeader('accept', 'application/json')
-      // request.setRequestHeader('accept', '*/*')
-      if (method === 'POST') {
-        request.setRequestHeader('Content-Type', `application/json`)
-        request.send(JSON.stringify(payload))
-      } else {
-        request.setRequestHeader('Authorization', `Bearer ${payload}`)
```
Comment (zachferland) [inline src/utils/index.js]: Token not passed in header anymore?

Comment (oed) [reply src/utils/index.js]: Yeah, I'm not sure why we did that. Don't think the address-server checks that, but we can double check.

**Comment by oed** [comment]
Do you think we should switch to cross-fetch in this PR or hold of for now?

**Comment by zachferland** [comment]
nah, not necessary in this pr

**Pull request merged**